src/app/app.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from '@angular/core';
+import { weatherData, widgetListInterface, widgetWeatherData } from './interfaces';
+
+import { WeatherClientService } from './weather-client.service';
+import { AppComponentService } from './app-component.service';
+
+
+interface cityOption {
```

Comment:
Please use PascalCase for interfaces/types/enums so it would be easy to recognize what is it, variable or type alias.